companies/v-partei.json

```diff
@@ -7,12 +7,13 @@
         "political party"
     ],
     "name": "Partei für Veränderung, Vegetarier und Veganer (V-Partei³)",
-    "address": "Hochfeldstraße 4\n86159 Augsburg\nDeutschland",
+    "address": "Schönbachstr. 4a\n86154 Augsburg\nDeutschland",
     "phone": "+49 821 29716264",
-    "email": "info@harle.de",
+    "email": "info@v-partei.de",
```

Comment:
I'm going to revert this change. The privacy policy still lists info@harle.de as the DPO email and says "Jede betroffene Person kann sich jederzeit bei allen Fragen und Anregungen zum Datenschutz direkt an unseren Datenschutzbeauftragten wenden."